### PR TITLE
Fix Codecov by ignoring Tests folder recursively

### PR DESCRIPTION
**Pull request by DominikBucher12**
Refer to: https://docs.codecov.io/docs/ignoring-paths 

and our internal conversation about code coverage

### CODE REVIEW

**Comment by nuno-vieira** [comment]
Maybe a better commit message? Like "Remove all files from test folder in codecov", something like this ?

**Review by VojtaStavik — APPROVED** [review]
Thanks 😎

File: codecov.yml [hunk 1]
```diff
@@ -4,5 +4,6 @@ ignore:
   - "**/*_Mock.swift"
   - "**/*_Vendor.swift"
   - "**/Generated/*.swift"
+  - "stream-chat-swift-docs/**/*"
```
Comment (VojtaStavik) [inline codecov.yml:7]: Why is this needed? 🤔

Comment (DominikBucher12) [reply codecov.yml:7]: https://getstream.slack.com/archives/CRML30N0H/p1617104227241200?thread_ts=1617102442.236200&cid=CRML30N0H

Comment (b-onc) [reply codecov.yml:7]: Docs are not tested :D My doc PR reported decrease in coverage

**Review by b-onc — APPROVED** [review]
